packages/benchmark/src/commands/list.ts

```diff
@@ -0,0 +1,21 @@
+import { Command } from '@oclif/core';
+import { TestScenarioLoader } from '@/testScenario/testScenarioLoader';
+import { loadConfig } from '@/config/config';
+
+export default class ListCommand extends Command {
+	static description = 'List all available test scenarios';
```

Comment:
Not a fan of the naming `test scenario`, sounds like integration or e2e testing. Maybe we can define what a `benchmark task` or `benchmark case` or `benchmark scenario` is in the README and in the code simply refer to it as `task` or `case` or `scenario`.

Reply:
I guess the full name would be `performance test scenario`. `Benchmark` and `performance test` are used pretty much interchangeable here. I believe `benchmark scenario` makes most sense here.